components/cyclebutton/stories/template.js

```diff
@@ -2,20 +2,54 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
 // import { ifDefined } from 'lit-html/directives/if-defined.js';
 
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js"
```

Comment:
```suggestion
import { Template as Icon } from "@spectrum-css/icon/stories/template.js";
import { Template as ActionButton } from "@spectrum-css/actionbutton/stories/template.js";
```

Rather than re-define the markup below, we should pull in the template from ActionButton's story so that we can leverage that. That keeps our source of truth separate so if ActionButton's markup changes, we'll inherit that here automatically.